config-sample-ent-drive.yaml

```diff
@@ -4,6 +4,7 @@ drive:
 execute: # All tests run.
   - fw_update_simple
   - opal_capable
+  #- opal_block_sid # Test Block SID Authentication function
```

Comment:
Probably indicate to read the README.md if you want to enable this test.

Reply:
thanks, updated this to link to readme.